src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,166 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
```

Comment:
do we need constructor without parameters?

Reply:
fixed